Data/FE15/Types/Sprite.yml

```diff
@@ -0,0 +1,90 @@
+---
```

Comment:
No change after review,
But still uses static value for entry count for testing purposes.